blueprints/addon/index.js

```diff
@@ -283,8 +283,6 @@ module.exports = {
     '^config.*': 'tests/dummy/:path',
     '^public.*': 'tests/dummy/:path',
 
-    '^addon-config/ember-try.js': 'tests/dummy/config/ember-try.js',
```

Comment:
`'^config.*': 'tests/dummy/:path',` should cover this as well. Tests should verify.